source/helpers/hotfix.ts

```diff
@@ -22,6 +24,21 @@ export const updateHotfixes = cache.function(async (): Promise<string[][]> => {
 	cacheKey: () => 'hotfixes',
 });
 
+export const updateStyleHotfixes = cache.function(async (version: string): Promise<string> => {
+	// We can't use `https://raw.githubusercontent.com` because of permission issues https://github.com/refined-github/refined-github/pull/3530#issuecomment-691595925
+	const request = await fetch(`https://api.github.com/repos/refined-github/refined-github/contents/style/${version}.css?ref=hotfix`);
+	const {content} = await request.json();
+
+	if (!content) {
+		return '';
+	}
+
+	return atob(content).trim();
+}, {
+	maxAge: {hours: 6},
+	cacheKey: ([version]) => 'style-hotfixes:' + version,
```

Comment:
No need to use the version, please match the existing hotfix usage. Also the style hotfix should be cleared when a new version is installed (like the regular hotfixes)